downstream/modules/platform/ref-ports-firewalls-requirements.adoc

```diff
@@ -17,16 +17,18 @@ The default destination ports and installer inventory listed below are configura
 .PostgreSQL
 [options="header"]
 |===
-|Port |Protocol |Service |Installer Inventory Variable |Required for
+|Port |Protocol |Service |Direction |Installer Inventory Variable |Required for
```

Comment:
```suggestion
|Port |Protocol |Service |Direction |Installer Inventory Variable |Required for
```
```suggestion
|Port |Protocol |Service |Direction |Installer Inventory Variable |Usage
```

Reply:
Global comment here but I won't make the suggestion for each occurrence. "Required for" seems odd to me like a dangling or unfinished phrase. When I read that it seems like there is text missing but maybe it's more obvious in the rendered table.

Reply:
Yes, I understand that it seems unfinished. However, that is the wording I saw in other tables across our CCS guides  as well as google docs created by the SMEs and stakeholders. If this is the language that they are using, then I want to keep it. Appreciate this comment.